JS/Lesson24/goksel/style.css

```diff
@@ -0,0 +1,142 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+body {
+    background-color: #000;
+}
+
+.form-container {
+    background-color: #fff;
+    border-radius: 9px;
+    width: 900px;
+    margin: 75px auto;
+}
+
+/* .form is for to remove content */
+.form {
+    display: none;
+}
+
+/* .active-form is for to make active content of current form */
+.active-form {
```

Comment:
It would make more sence to name this class just 'form-container'. Then use class 'hidden' when you need to hide elements.